### DISCUSSION TITLE
How do I declare multi-line strings in Zephir

### DISCUSSION THREAD

**Discussion by henrywood** [zephir-lang/zephir#2414] [General]
How do I declare a multi-line string in Zephir ?

I have this member in a PHP class:
         private static string     $RemoveHyphensRegex      =  "#
             (
              -
              [ \t]* ( (\r\n) | \n | \r )+ [ \t\r\n]*
             )
             ([^ \t\r\n]+)
             \s*
    #msx";  
  
  Please help !
  
  Thanks !

**Comment by Jeckerson** [comment]
Hello, here are some examples:

```zephir
let variable = "
This is
multiline
string.
";
```

```zephir
let alu = "GGCCGGGCGCGGTGGCTCACGCCTGTAATCCCAGCACTTTGG" .
          "GAGGCCGAGGCGGGCGGATCACCTGAGGTCAGGAGTTCGAGA" .
	  "CCAGCCTGGCCAACATGGTGAAACCCCGTCTCTACTAAAAAT" .
	  "ACAAAAATTAGCCGGGCGTGGTGGCGCGCGCCTGTAATCCCA" .
	  "GCTACTCGGGAGGCTGAGGCAGGAGAATCGCTTGAACCCGGG" .
	  "AGGCGGAGGTTGCAGTGAGCCGAGATCGCGCCACTGCACTCC" .
	  "AGCCTGGGCGACAGAGCGAGACTCCGTCTCAAAAA";
```

**Reply by henrywood** [reply]
But this does not work:

private static RemoveHyphensRegex = "#
(
-
[ \t]* ( (\r\n) | \n | \r )+ [ \t\r\n]*
)
([^ \t\r\n]+)
\s*
#msx";

(when used in a ZEPHIR class) - The compiler just says "syntax error")

?

**Reply by henrywood** [reply]
got it to work
thanks
